package.json

```diff
@@ -7,9 +7,17 @@
     "start": "DEBUG=ERROR-*,WARN-*,INFO-* node build/bot",
     "dev": "DEBUG=ERROR-*,WARN-*,INFO-*,DEBUG-* nodemon src/bot.ts",
     "test": "jest",
-    "test-coverage": "jest --coverage && ls -la ./coverage && cat ./coverage/lcov.info | coveralls",
+    "test-coverage": "jest --coverage && cat ./coverage/lcov.info | coveralls",
+    "lint": "eslint src --ext .ts,.tsx,.js",
+    "lint:fix": "eslint --fix . --ext .ts,.tsx,.js",
+    "verify": "npm run lint && npm run test",
```

Comment:
Can you use `yarn` instead of `npm`?

Reply:
In the scripts is not a good idea, it would make it incompatible with npm

We use yarn officially for building. For running the code, it doesn't make a difference, so changing this would add no value and make it `yarn` only compatible